posthog/hogql_queries/experiments/experiment_trends_query_runner.py

```diff
@@ -226,7 +255,63 @@ def calculate(self) -> ExperimentTrendsQueryResponse:
 
         def run(query_runner: TrendsQueryRunner, result_key: str, is_parallel: bool):
             try:
-                result = query_runner.calculate()
+                database = create_hogql_database(team_id=self.team.pk)
+                if self._is_data_warehouse_query(query_runner.query):
+                    series_node = cast(DataWarehouseNode, query_runner.query.series[0])
+                    table = database.get_table(series_node.table_name)
+                    table.fields["events"] = LazyJoin(
+                        from_field=[series_node.distinct_id_field],
+                        join_table=database.get_table("events"),
+                        join_function=lambda join_to_add, context, node: (
+                            ast.JoinExpr(
+                                table=ast.SelectQuery(
```

Comment:
This is currently joining over all events, but we should join only over the `$feature_flag_events`. We probably need a `where` clause here.

In the future we should also support custom exposure events here, but for now feel free to hardcode `$feature_flag_called` and leave a comment.